linux_os/guide/system/network/network-iptables/iptables_ruleset_modifications/set_iptables_outband_n_established_connections_configured/rule.yml

```
@@ -0,0 +1,47 @@
+documentation_complete: true
+
+title: 'Ensure outbound and established connections are configured'
+
+description: |-
+    Configure the firewall rules for new outbound, and established connections.
```

```suggestion
    Configure the firewall rules for new outbound and established connections.
```
You don't really need the comma here. Same for the other ocurrences